config/pitest-suppressions/pitest-filters-suppressions.xml

```diff
@@ -216,6 +216,24 @@
     <lineContent>tagMessageRegexp = null;</lineContent>
   </mutation>
 
+  <mutation unstable="false">
+    <sourceFile>SuppressWithNearbyTextFilter.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.filters.SuppressWithNearbyTextFilter</mutatedClass>
+    <mutatedMethod>accept</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.NonVoidMethodCallMutator</mutator>
+    <description>removed call to java/lang/String::equals</description>
+    <lineContent>if (!currentFileAbsolutePath.equals(eventFileTextAbsolutePath)) {</lineContent>
+  </mutation>
+
+  <mutation unstable="false">
+    <sourceFile>SuppressWithNearbyTextFilter.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.filters.SuppressWithNearbyTextFilter</mutatedClass>
+    <mutatedMethod>accept</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.RemoveConditionalMutator_EQUAL_IF</mutator>
+    <description>removed conditional - replaced equality check with true</description>
+    <lineContent>if (!currentFileAbsolutePath.equals(eventFileTextAbsolutePath)) {</lineContent>
```

Comment:
https://github.com/checkstyle/checkstyle/issues/12747#issuecomment-1435769940 this comment suggested that we can kill this suppression by ensuring that `collectSuppressions()` is called only once. 

But the method is `private`. I see `mockito` is available but verifying private methods is not supported it seems:
https://stackoverflow.com/a/8799482/9553927

I see `TestUtil` has some methods that could help but not sure how yet.

Reply:
@rnveach I am clueless as to how this mutation can be resolved.